client/test/actions/user.spec.js

```diff
@@ -0,0 +1,218 @@
+import moxios from 'moxios';
+import thunk from 'redux-thunk';
+import configureMockStore from 'redux-mock-store';
+import UserActions from '../../actions/user';
+
+const middleware = [thunk];
+const mockStore = configureMockStore(middleware);
+
+describe('User Actions', () => {
+  beforeEach(() => moxios.install());
+  afterEach(() => moxios.uninstall());
+
+  describe('Signup Action', () => {
+    it('Should make a post request to sign in users', (done) => {
+      moxios.stubRequest('https://postit-api.herokuapp.com/api/user/signup', {
```

Comment:
I think it will be ideal to have the api host in a variable. So, in case the url changes, the work to be done would be less as you would not have to replace in every places where you have the former url 
Maybe something like this.

```
describe('Signup Action', () => {
    const apiUrl = 
    it('Should make a post request to sign in users', (done) => {
      moxios.stubRequest(`${apiUrl}/api/user/signup`, {

     })
    })
})